docs/index.md

```diff
@@ -56,7 +56,7 @@ from typing import Optional
 class AddressFilter(Filter):
     street: Optional[str]
     country: Optional[str]
-    city__in: Optional[list[str]]
+    city__in: Optional[List[str]]
```

Comment:
I regret my decision when we started supporting 3.9 to update the docs with compatible code. I think that the documentation should show the syntax for the latest supported version. And if I have time to figure out how to show different code examples for different python versions then we can add these. WDYT?